scene/2d/cpu_particles_2d.cpp

```diff
@@ -597,7 +597,7 @@ void CPUParticles2D::_validate_property(PropertyInfo &p_property) const {
 	}
 
 	if (p_property.name == "seed" && !use_fixed_seed) {
```

Comment:
Unrelated now, but it's better like this.
```suggestion
	if (!use_fixed_seed && p_property.name == "seed") {
```